nautobot/utilities/templatetags/helpers.py

```diff
@@ -304,32 +304,27 @@ def percentage(x, y):
 
 @library.filter()
 @register.filter()
-def get_docs(model):
-    """Render and return documentation for the specified model.
+def get_docs_url(model):
```

Comment:
This is technically a breaking change (since you're removing `get_docs` as an available filter), but I think I'm okay with it as I can't think of a case where a plugin would be using `get_docs` presently.

Reply:
True, and if you would like, I can add that back in with a depreciation warning.

Reply:
I was just thinking about this, and the whole reason for this PR is to fix plugins not working with this. It is possible that someone changes docs_root to another directory and adds their plugin docs to it. I guess whatever you want.